lookout/style/format/descriptions.py

```diff
@@ -104,8 +118,8 @@ def describe_sample_ordinal(feature: OrdinalFeature, values: ndarray) -> str:
 
 
 @singledispatch
-def describe_rule_splits(feature: BagFeature, name: str,
-                         splits: List[Tuple[bool, floating, int]]) -> str:
+def _describe_rule_splits(feature: BagFeature, name: str,
```

Comment:
We should move it to FE class eventually because it is tightly coupled and I don't like playing C 😉